liquid-staking/tests/test.rs

```diff
@@ -174,44 +179,44 @@ fn liquid_staking_multiple_operations() {
     sc_setup.check_user_egld_balance_denominated(sc_setup.sc_wrapper.address_ref(), 0u128);
 
     sc_setup.add_liquidity(&first_user, 50u64);
-    sc_setup.check_delegation_contract_values_denominated(
-        &delegation_contract1,
-        63027397260273972600u128,
-    );
-    sc_setup.update_staking_contract_params(
-        &sc_setup.owner_address.clone(),
-        &delegation_contract1,
-        1061,
-        1100,
-        3,
-        10_000u64,
-    );
-
-    sc_setup.add_liquidity(&second_user, 40u64);
-    sc_setup.check_delegation_contract_values_denominated(
-        &delegation_contract1,
-        63027397260273972600u128,
-    );
-    sc_setup.update_staking_contract_params(
-        &sc_setup.owner_address.clone(),
-        &delegation_contract1,
-        1090,
-        1100,
-        3,
-        10_000u64,
-    );
-
-    sc_setup.add_liquidity(&third_user, 30u64);
-    sc_setup.check_delegation_contract_values(&delegation_contract3, 100u64);
-    sc_setup.update_staking_contract_params(
-        &sc_setup.owner_address.clone(),
-        &delegation_contract3,
-        1100,
-        1100,
-        3,
-        9_000u64,
-    );
-
+    /*  sc_setup.check_delegation_contract_values_denominated(
+            &delegation_contract1,
+            63027397260273972600u128,
+        );
+        sc_setup.update_staking_contract_params(
+            &sc_setup.owner_address.clone(),
+            &delegation_contract1,
+            1061,
+            1100,
+            3,
+            10_000u64,
+        );
```

Comment:
can we have these back ?

Reply:
we cannot work with delegation and governance in blackbox as fas as I know from what I've been told